src/core/settings.py

```
@@ -100,6 +100,40 @@ class Settings(BaseSettings):
+    @property
+    def POSTGRES_URI(self) -> str:
+        """PostgreSQL connection URI for LangGraph checkpointer.
+
+        .env 에 정의된 DB_HOST, DB_PORT, DB_NAME, DB_USERNAME, DB_PASSWORD 값을 기반으로
+        LangGraph용 접속 URI를 동적으로 생성한다. 실제 자격 증명 값은 .env 에만 존재하고
+        코드에는 하드코딩되지 않는다.
+        """
+        if not self.DB_USERNAME or not self.DB_PASSWORD:
+            raise ValueError(
+                "DB_USERNAME and DB_PASSWORD must be set in environment (.env)"
+            )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)